force-app/main/default/classes/TaskTriggerHandler.cls

```diff
@@ -0,0 +1,104 @@
+public without sharing class TaskTriggerHandler {
+  @testvisible
+  private static final Id recordTypeUS = Schema.SObjectType.Account.getRecordTypeInfosByDeveloperName()
+    .get('United_States')
+    .getRecordTypeId();
+
+  public static void handleAfterInsert(List<Task> newTasks) {
```

Comment:
handleAfterInsert and the other names are naming conventions for trigger patterns

I see that you copied my method names by the letter - thank you but that is not a pre-requesite.